src/Compiler/CMakeLists.txt

```diff
@@ -29,10 +24,56 @@ add_custom_target(ExternalUtil DEPENDS ${FILE_GENERATE_DIR}/ExternalUtil.hpp)
 
 get_property(OMLibs GLOBAL PROPERTY ONNX_MLIR_LIBS)
 
-add_onnx_mlir_library(CompilerUtils STATIC
-  CompilerUtils.cpp
+add_onnx_mlir_library(CompilerOptions
+  CompilerOptions.cpp
 
-  EXCLUDE_FROM_OM_LIBS
```

Comment:
Shouldn't this lib and `CompilerPasses` be excluded from `OMLibs`? Then you can use `OMLibs` for linking below instead of having to list them all?

Reply:
They should definitely be excluded.

Reply:
Added EXCLUDE_FROM_OM_LIBS. Thanks!